app/views/forms/odk_manifest.xml.erb

```diff
@@ -7,5 +7,13 @@
         <downloadUrl><%= url_for_path(@ifa.path) %></downloadUrl>
       </mediaFile>
     <% end %>
+
+    <% @questions.each do |q| %>
+      <mediaFile>
+        <filename><%= q.scramble_audio_prompt_filename %></filename>
+        <hash><%= Digest::MD5.file(q.audio_prompt.path).hexdigest %></hash>
```

Comment:
Nice! This should also go in an ODK decorator. Perhaps `q.audio_prompt_md5`.

Reply:
👍